server/database/queries.js

```diff
@@ -0,0 +1,151 @@
+const pgp = require('pg-promise')();
+
+module.exports = {
+  getUser: function (userId) {
+    return db.query('select * from users');
+  },
+  getUsers: function () {
+    return db.query('select * from users');
+  },
+  getUserOwnedAuctions: function (userId) {
+    return db.query('select * from auctions where owner_id=$1', [userId]);
+  },
+  getUserBiddingAuctions: function () {
+    //TODO
+  },
+  getUserBids: function (userId) {
+    return db.query('select * from bids where bidder_id=$1', [userId]);
+  },
+  getUserBidsPerAuction: function ({user_id, auction_id}) {
+    /*
+    {
+      user_id:
+      auction_id:
+    }
+    select * from bids where bidder_id=1 and auction_id=1
+    */
+    return db.query('select * from bids where bidder_id=$1 and auction_id=$2', [user_id, auction_id]);
+  },
+  getUserMessages: function (userId) {
+    
+  },
+  getAuctions: function () {
+    return db.query('select * from auctions');
+  },
+
+  createUser: function (userObj) {
+    /*
+    {
+      password:
+      username:
+      first_name:
+      last_name:
+      address:
+      email:
+      type:
+    }
+    insert into users (username, first_name, last_name, email, address, type, password) values ('wdku', 'winston', 'ku', 'wdku5791@gmail.com', '471 quail drive', 'user', 'hashed')
+    */
+    return db.query('insert into users \
+      (username, first_name, last_name, email, address, type, password)\
+      values \
+      (${username}, ${first_name}, ${last_name}, ${email}, ${address}, ${type}, ${password})\
+      returning id', userObj);
```

Comment:
These ${} wrapped values probably need to be wrapped into single quotes to work.